gitlab/pkg/reconciler/source/controller.go

```diff
@@ -20,20 +20,22 @@ import (
 	"context"
 
 	"github.com/kelseyhightower/envconfig"
+	"knative.dev/eventing/pkg/reconciler/source"
 
 	//k8s.io imports
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/tools/cache"
 
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	serviceclient "knative.dev/serving/pkg/client/injection/client"
+	kserviceinformer "knative.dev/serving/pkg/client/injection/informers/serving/v1/service"
+
```

Comment:
The comments below don't make any more sense after this new grouping, so they should be removed (or the grouping should be kept according to comments).

Reply:
i think this was done by my formatter, let me remove it